Week3/solution.txt.txt

```diff
@@ -0,0 +1,216 @@
+Write a python script to perform CRUD operation on following table of "ESM.db" database.
+
+Employee ( eid, ename, dept, basic, branch). eid must be primary key.
+
+Department : Account, Inventory, IT, HR.
+
+Peform Following Operation.
+
+1. Create table
+Ans:-
+
+import sqlite3 as sql
+con=sql.connect("c://sqlite3//ESM.db")
+#create cursor
+cur=con.cursor()
+table="create table employee\
+( \
+    eid number primary key,\
+    ename text,\
+    dept text,\
+    basic number,\
+    branch text\
+    )"
+cur.execute(table)
+con.commit()
+
+2. Insert 5 Records directly, 5 records using tuple and 5 records using taking input from user
+Ans:-
+
+import sqlite3 as sql
+con=sql.connect("c://sqlite3//ESM.db")
+cur=con.cursor()
+ins="insert into employee\
+    values\
+    (101,'om','Account',25000,'surat'),\
+    (102,'sai','Inventory',28000,'vyara'),\
+    (103,'ram','IT',30000,'ghandhinagar'),\
+    (104,'pritam','HR',23000,'mumbai'),\
+    (105,'krishna','Inventory',25000,'baroda')"
+cur.execute(ins)
+con.commit()
+'''
++-----+---------+-----------+-------+--------------+
+| eid |  ename  |   dept    | basic |    branch    |
++-----+---------+-----------+-------+--------------+
+| 101 | om      | Account   | 25000 | surat        |
+| 102 | sai     | Inventory | 28000 | vyara        |
+| 103 | ram     | IT        | 30000 | ghandhinagar |
+| 104 | pritam  | HR        | 23000 | mumbai       |
+| 105 | krishna | Inventory | 25000 | baroda       |
++-----+---------+-----------+-------+--------------+
+'''
+
+#record input using tuple
+record= [
+        (106,'Krunal','HR',21000,'Bardoli'),
+        (107,'Radhe','Account',25000,'sarbhon'),
+        (108,'Rajesh','IT',26500,'Navsari'),
+        (109,'Dhruv','Inventory',20000,'Bardoli'),
+        (110,'Grijesh','HR',23000,'surat')
+        ]
+cur.executemany("insert into employee values(?,?,?,?,?)",record)
+con.commit()
+
+'''
++-----+---------+-----------+-------+---------+
+| eid |  ename  |   dept    | basic | branch  |
++-----+---------+-----------+-------+---------+
+| 106 | Krunal  | HR        | 21000 | Bardoli |
+| 107 | Radhe   | Account   | 25000 | sarbhon |
+| 108 | Rajesh  | IT        | 26500 | Navsari |
+| 109 | Dhruv   | Inventory | 20000 | Bardoli |
+| 110 | Grijesh | HR        | 23000 | surat   |
++-----+---------+-----------+-------+---------+
+'''
+
+#record insert using user input
+ins2="insert into employee values(?,?,?,?,?)"
+l=[ ] #create empty list
+for i in range(0,5):
+    n=int(input('Enter Employee Id: '))
+    name=input('Enter Employee Name: ')
+    dept=input('Enter Department Name: ')
+    basic=int(input('Enter Basic: '))
+    branch=input('Enter Branch: ')
+    t=[n,name,dept,basic,branch]
+    l.append(t)
+cur.executemany(ins2,l)
+con.commit()
+'''
++-----+---------+-----------+-------+--------+

```

Comment:
i want output in a box form the what can i do?

Reply:
use  .mode box  command